### PR TITLE
Feature/265 standard page FE

### PR DESCRIPTION
**Pull request by BrianXu20**
[Ticket #265](https://projects.torchbox.com/projects/tbxcom/tickets/265) is focus on Standard Page. 
In order to make Standard Page work, I have to do the following modification:

- Customize styles of all the blocks under StoryBlock model.

- Fix the template location for standard page

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/aligned_image_block.yaml [hunk 1]
```diff
@@ -7,5 +7,4 @@ context:
 tags:
   image:
     value.image original:
-      raw: |
-        <img src="http://via.placeholder.com/1000x300" alt="Some alt">
+      raw: <img src="http://via.placeholder.com/1000x300" alt="Some alt">
```
Comment (SharmaineLim) [inline tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/aligned_image_block.yaml:10]: Note: Either way works, so this is fine.

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/bustout_block.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% load wagtailimages_tags %}
+<div>
+    {% image value.image original %}
+    {{value.text}}
```
Comment (SharmaineLim) [inline tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/bustout_block.html:4]: I think this is rich text.

File: tbx/core/blocks.py [hunk 3]
```diff
@@ -87,13 +95,27 @@ class StoryBlock(StreamBlock):
...
+        template="patterns/molecules/streamfield/blocks/wide_image_block.html",
+    )
+    bustout = BustoutBlock(template="patterns/molecules/streamfield/blocks/bustout_block.html")
+    pullquote = PullQuoteBlock(template="patterns/molecules/streamfield/blocks/pullquote_block.html")
+    raw_html = RawHTMLBlock(
+        label='Raw HTML',
+        icon="code",
+        template="patterns/molecules/streamfield/blocks/raw_html_block.html",
+    )
     embed = EmbedBlock(icon="code")
```
Comment (SharmaineLim) [inline tbx/core/blocks.py:113]: I'm mistaken, please also create a block for EmbedBlock.

File: tbx/core/blocks.py [hunk 4]
```diff
@@ -87,13 +95,27 @@ class StoryBlock(StreamBlock):
...
-    markdown = MarkdownBlock(icon="code")
+    markdown = MarkdownBlock(
+        icon="code",
+        template="patterns/molecules/streamfield/blocks/raw_html_block.html",
```
Comment (SharmaineLim) [inline tbx/core/blocks.py:116]: Doesn't quite look right. :)

Comment (BrianXu20) [reply tbx/core/blocks.py:116]: cons of copt pasted XD

**Review by SharmaineLim — APPROVED** [review]
Please run `black tbx` in vagrant and `npm run format` on your machine to fix the failing tests. Other than that, this looks good.

**Pull request merged**